src/db/addTermMgmt.sql

```diff
@@ -230,9 +230,8 @@ CREATE OR REPLACE FUNCTION getTermInstructorCount(termID INT)
 RETURNS INT
 AS
 $$
-BEGIN
-   RAISE WARNING 'Function not implemented';
-END
+   SELECT COUNT(*) 
+   FROM (SELECT DISTINCT Instructor1, Instructor2, Instructor3 FROM Section);
```

Comment:
This seems to be selecting a distinct combination of instructors, which means that some instructors will be counted multiple times, and some may be skipped. Will provide suggestions later.